arm/Microsoft.Storage/storageAccounts/deploy.bicep

```diff
@@ -152,7 +152,7 @@ var diagnosticsMetrics = [for metric in diagnosticMetricsToEnable: {
 }]
 
 var maxNameLength = 24
-var uniqueStorageNameUntrim = '${uniqueString('Storage Account${basetime}')}'
+var uniqueStorageNameUntrim = uniqueString('Storage Account${basetime}')
```

Comment:
Not sure I fully agree to the naming here. This is used in case name is not provided I take it. If this name is generated like this, it will not be idempotent. Next time it runs, a new SA will be created.
```suggestion
```

Reply:
I guess that was the idea. Anyhow - I strongly vote against changing this in this PR. We have a separate issue for this topic open here #1443.

The only change I did as part of this PR was to update the string-builder as per the Bicep warning.

Reply:
Agree